util/cbmem/cbmem.c

```diff
@@ -116,8 +122,8 @@ static const void *map_memory(struct mapping *mapping, unsigned long long phys,
 			phys - mapping->offset);
 
 	if (v == MAP_FAILED) {
-		debug("Mapping failed %zuB of physical memory at 0x%llx.\n",
-			mapping->virt_size, phys - mapping->offset);
+		debug("Mapping failed %zuB of physical memory at 0x%llx.\n Error: %s\n",
+			mapping->virt_size, phys - mapping->offset, strerror(errno));
```

Comment:
Have you checked if `strerror(errno)` is available on BSD systems?

Reply:
Should be available as it is documented.
* [strerror doc](https://www.freebsd.org/cgi/man.cgi?query=strerror&apropos=0&sektion=0&manpath=FreeBSD+6.2-RELEASE&format=html)
* [errno doc](https://www.freebsd.org/cgi/man.cgi?query=errno&sektion=2&manpath=freebsd-release-ports

Additionally this function is already used in cbmem [here](https://github.com/Dasharo/coreboot/blob/cc39cf71072a4da2afa828d30e0031fba46f95be/util/cbmem/cbmem.c#L450).

Reply:
Right, but at that point it is protected with `#if defined(linux)`. We'll have to see if CI likes it or not.